priv/repo/migrations/20200724162310_add_displayname_to_user_auth_table.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Glimesh.Repo.Migrations.AddDisplaynameToUserAuthTable do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :displayname, :string, default: nil
+    end
+    execute "UPDATE users SET displayname = username"
+    execute "UPDATE users SET username = lower(username)"
```

Comment:
Why do we require usernames be lowercase? I personally like having capital letters in my URI - we can achieve case insensitive URIs without requiring all usernames be all lowercase.

Reply:
If I'm in the minority here then that's fine, but I think it's worth discussing 😁

Reply:
This is why displaynames were created, for capital letters. Read the issue this is tied to.